### DISCUSSION TITLE
Support open source google fonts for labels

### DISCUSSION THREAD

**Discussion by yvonneekanim** [specify/specify7#5524] [Feature Requests]

**Comment by benanhalt** [comment]
Google fonts: https://fonts.google.com/

There are more than 900 families available, so we would need to decide which ones to package.

Or, maybe make a tool that can generate a font jar from a download from Google fonts.

**Comment by maxpatiiuk** [comment]
I suppose there is no way to dynamically link to a font like one might do in CSS?

**Comment by grantfitzsimmons** [comment]
> @tlammer
> will not fix. requires documentation on which fonts can be used and how to add new fonts to list of available fonts.

> @benanhalt 
> Instructions are at https://github.com/specify/report-runner-service#fonts. I sent Norine the list of fonts currently installed on the cloud.

Going to look into a way to add the current set of Google fonts onto the cloud.
